docs/concepts/events.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 6
 
 # Events
 
-Flow events are special values that can be emitted during the execution of a Cadence program. 
+Flow events are special values that are emitted to off-chain chain observers during the execution of a Cadence program.
```

Comment:
they are not really emitted to off-chain observers, they are emitted on the network, but off-chain observers can observe them. There's no push mechanism for off-chain observer to receive it.